dotenvx_runtime.py

```
@@ -0,0 +1,69 @@
+    loaded: Dict[str, str] = {}
+    for line in env_path.read_text().splitlines():
+        if line.strip().startswith("#") or "=" not in line:
+            continue
+        key, val = line.split("=", 1)
+        key = key.strip()
+        val = val.strip()
+        if private_key:
+            try:
+                val = _decrypt(val, private_key)
+            except Exception:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)